test-utils/loadtester/src/main.rs

```diff
@@ -29,41 +39,160 @@ fn configure_logging(log_level: log::LevelFilter) {
 }
 
 fn main() {
-    configure_logging(log::LevelFilter::Debug);
-    let genesis_config = GenesisConfig::testing_spec(400, 10);
-    let accounts: Vec<_> = genesis_config.records[0]
-        .iter()
-        .filter_map(|r| match r {
-            StateRecord::Account { account_id, .. } => Some(account_id.clone()),
-            _ => None,
-        })
-        .collect();
-
-    let addrs = [
-        "35.236.106.188:3030",
-        "35.235.115.64:3030",
-        "35.235.75.161:3030",
-        "35.236.113.178:3030",
-        "35.236.42.186:3030",
-        "35.236.29.55:3030",
-        "35.235.84.221:3030",
-        "35.236.44.50:3030",
-        "35.236.84.38:3030",
-        "35.236.37.104:3030",
-    ];
-
-    let num_nodes = addrs.len();
+    let version =
+        Version { version: crate_version!().to_string(), build: git_version!().to_string() };
+    let default_home = get_default_home();
+
+    let matches = App::new("NEAR Protocol loadtester")
+        .version(format!("{} (build {})", version.version, version.build).as_str())
+        .arg(
+            Arg::with_name("v")
+                 .short("v")
+                 .multiple(true)
+                 .help("Sets the level of verbosity"))
+        .subcommand(SubCommand::with_name("run").about("Run loadtester")
+            .arg(
+                Arg::with_name("accounts")
+                    .long("accounts")
+                    .takes_value(true)
+                    .default_value("400")
+                    .help("Number of accounts to use"),
+            )
+            .arg(
+                Arg::with_name("prefix")
+                    .long("prefix")
+                    .takes_value(true)
+                    .default_value("near")
+                    .help("Prefix the account names (account results in {prefix}.0, {prefix}.1, ...)"),
+            )
+            // Will be able to self discover all available rpc endpoint once nearcore expose it.
```

Comment:
Why is this part commented out?

Reply:
Because #1320 is not merged, for now it's only possible to give all rpc socket addresses to test

Reply:
#1320 is merged now.

Reply:
still wait it into master. Hopefully after sync master into staging. or recreate this PR to staging.